website/docs/d/network_manager_network_group.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_network_manager_network_group"
+description: |-
+  Get information about a Network Manager Network Group.
+---
+
+# azurerm_network_manager_network_group
+
+Use this data source to access information about a Network Manager Network Group.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+data "azurerm_subscription" "current" {
+}
+
+resource "azurerm_network_manager" "example" {
+  name                = "example-network-manager"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  scope {
+    subscription_ids = [data.azurerm_subscription.current.id]
+  }
+  scope_accesses = ["Connectivity", "SecurityAdmin"]
+  description    = "example network manager"
+}
+
```

Comment:
Aren't we missing a `azurerm_network_manager_network_group` resource in this example?